core-bundle/contao/templates/twig/content_element/text.html.twig

```diff
@@ -1,29 +1,10 @@
 {% extends "@Contao/content_element/_base.html.twig" %}
-{% use "@Contao/component/_figure.html.twig" %}
+{% use "@Contao/component/_text.html.twig" %}
 
-{% set content_layout_attributes = attrs(content_layout_attributes|default)
-    .addClass('media media--' ~ layout, layout and image)
-%}
-
-{# By default we add the layout attributes to the wrapper element. If you want to
- disable this behavior, set "default_content_layout" to false. #}
 {% set attributes = attrs(attributes|default)
-    .mergeWith(content_layout_attributes, default_content_layout is not defined or default_content_layout)
+    .addClass('media media--' ~ layout, layout and image)
 %}
```

Comment:
I think this needs to be reverted as this was an explicit extension point for extending templates.

It does not seem to be related to other changes that move the code to the *_text.html.twig* component, right?

Reply:
This seems pretty over-engineered to me, especially since the template now only consists of `{{ block('text_component') }}`. You should use the `_text.html.twig` component in your extension instead.

Reply:
I think extending the `text.html.twig` template is still a very valid use case, especially on the project level (e.g. for overwriting the default `text.html.twig` template). I don’t see any benefit of this BC break, and I think it is unrelated to the accordion element.

Reply:
The idea of the attributes is, that you can provide/define them without overwriting blocks. This is especially helpful for extensions that want to add attributes to logical places but don't care about the specific markup. Not saying that this *must* be like it was or that things are consistent everywhere, but removing a variable that was intentionally merged (`content_layout_attributes|default`) with one defined in the outer scope by someone else is potentially breaking stuff.

(Just a quick explanation, I unfortunately didn't have time to review this and the other Twig related PRs properly, yet… :upside_down_face:)

Reply:
Reverted in 2b028470e5bec31618e9af80ccc480c025488e14.